exposed-kotlin-datetime/src/test/kotlin/org/jetbrains/exposed/sql/kotlin/datetime/KotlinTimeTests.kt

```diff
@@ -91,15 +91,33 @@ open class KotlinTimeBaseTest : DatabaseTestsBase() {
     }
 }
 
-fun assertEqualDateTime(d1: LocalDateTime?, d2: LocalDateTime?) {
+fun <T> assertEqualDateTime(d1: T?, d2: T?) {
     when {
         d1 == null && d2 == null -> return
         d1 == null -> error("d1 is null while d2 is not on ${currentDialectTest.name}")
         d2 == null -> error("d1 is not null while d2 is null on ${currentDialectTest.name}")
-        else -> {
-            assertEquals(d1.toJavaLocalDateTime().toEpochSecond(ZoneOffset.UTC), d2.toJavaLocalDateTime().toEpochSecond(ZoneOffset.UTC), "Failed on epoch seconds ${currentDialectTest.name}")
+        d1 is LocalTime && d2 is LocalTime -> {
+            assertEquals(d1.toSecondOfDay(), d2.toSecondOfDay(), "Failed on seconds ${currentDialectTest.name}")
+            if (d2.nanosecond != 0) {
+                assertEqualFractionalPart(d1.nanosecond, d2.nanosecond)
```

Comment:
I would also add a message hrere

Reply:
@e5l As a parameter to `assertEqualFractionalPart` function you mean?

Reply:
yep

Reply:
@e5l Every case (for every DB dialect) within that function already has its own message. What other message would we add?